roles/base/templates/run-ansible-pull.sh.j2

```diff
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
```

Comment:
Yes. It's using `[[ ... ]]` in two places now. But, the variables being used are integers so word-splitting isn't an issue and both use comparison operators compatible with `test`.